engine/src/main/java/com/odysseusinc/arachne/executionengine/execution/r/DockerService.java

```diff
@@ -54,6 +63,9 @@ public class DockerService extends RService implements AutoCloseable {
     @Value("${docker.image.default}")
     private String defaultImage;
 
+    @Value("${docker.image.filter}")
+    private String filterRegex;
```

Comment:
It would be nice to have a default value for dev env

Reply:
This is a control input. If it is not specified, scanning is disabled. This is required to ensure existing tarball configurations stay as it is.. Documentation to be added to release notes.